Modules/getaddrinfo.c

```diff
@@ -351,6 +351,9 @@ getaddrinfo(const char*hostname, const char*servname,
             struct servent *sp;
             const char *proto;
 
+            if (ai->ai_flags & AI_NUMERICSERV)
```

Comment:
PEP 7: Please add { ... } braces around the return for new code like this one.

Reply:
Added the braces. I also noticed, that this should have used `ERR(EAI_NONAME);` instead of `return EAI_NONAME;` to set the res to NULL similar to the other error conditions.